style.css

```diff
@@ -178,8 +181,8 @@ a:hover {
 /* images */
 
 .section2-picture {
-    height: 15%;
-    width: 70%;
+    height: auto;
+    width: 100%;
```

Comment:
Good - always set _either_ the width _or_ the height of an image. Leaving the other to `auto` works perfectly to preserve the aspect ratio 👍